renderlib/CCamera.cpp

```diff
@@ -238,7 +208,11 @@ cameraManipulation(const glm::vec2 viewportSize, Gesture& gesture, CCamera& came
     glm::vec3 rotated_v = camera.m_From - camera.m_Target;
     glm::vec3 v = camera.m_From - camera.m_Target;
     glm::vec3 newEye, newUp;
-    trackball(drag.x, -drag.y, camera, newEye, newUp);
+
+    glm::quat q = trackball(drag.x, -drag.y, camera.m_From - camera.m_Target, camera.m_Up, camera.m_U);
+    newUp = q * camera.m_Up;
+    newEye = q * rotated_v + camera.m_Target;
+
     rotated_v = newEye - camera.m_Target;
```

Comment:
Does adding and then removing `camera.m_Target` do anything here?

Reply:
nope! I wondered if anyone would notice 😁

Reply:
This is now resolved in one hopefully last commit on this PR